packages/__docs__/src/Figure/index.js

```diff
@@ -77,6 +77,17 @@ class Figure extends Component {
     children: Children.oneOf(['FigureItem'])
   }
 
+  static allowedProps = [
+    'makeStyles',
+    'styles',
```

Comment:
These shouldn't be allowedProps, it is always a bad idea to pass the styles to another component.